moda-contracts/src/interfaces/IManagement.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.21;
+
+interface IManagement {
+    /// @dev Emitted when a manager is added by an artist
+    event ManagerAdded(address indexed artist, address indexed manager);
```

Comment:
Maybe we simplify the events to a single one. It makes filtering them easier.

`event ManagementPermissionSet(address indexed artist, address indexed manager, bool isManager);`